test/cypress/integration/widgets/deployments_view_spec.ts

```diff
@@ -897,7 +897,12 @@ describe('Deployments View widget', () => {
             useEnvironmentsWidget();
             cy.getSearchInput().clear().type(fullDeploymentName);
             getDeploymentsViewTable().contains(fullDeploymentName);
-            getDeploymentsViewTable().within(() => cy.get('[title="Click to drill down to subservices"]').click());
+            cy.get('.input.loading').should('not.exist');
+            getDeploymentsViewTable()
+                .contains(fullDeploymentName)
+                .parent()
+                .parent()
+                .within(() => cy.get('[title="Click to drill down to subservices"]').click());
```

Comment:
Something like this would make it even more robust as it would be less dependent on the HTML structure:
```suggestion
            getDeploymentsViewTable()
                .contains('tr', fullDeploymentName)
                .find('[title="Click to drill down to subservices"]').click();
```

Reply:
I also like the suggested direction 👍

Reply:
implemented, test are running - https://jenkins.cloudify.co/job/Stage-UI-System-Test/5041/ deployments_view_spec already passed with green ✔️